requirements.txt

```diff
@@ -1,9 +1,11 @@
 # This file is automatically generated by scripts/dev/recompile_requirements.py
 
 attrs==18.2.0
+bumpversion==0.5.3
```

Comment:
That'd mean everyone installing qutebrowser also needs to install bumpversion/lxml. Right now there's nothing like `requirements-dev.txt` (only `misc/requirements/*`, but it doesn't really fit anywhere), so I'd propose just removing those.

Reply:
Removed as suggested :).